ICM: icm-as ct lint failes

```
Linting chart 'icm-as => (version: "0.3.8", path: "charts/icm-as")'
Validating /home/runner/work/helm-charts/helm-charts/charts/icm-as/Chart.yaml...
Validation success! 👍
Error: [comments] missing starting space in comment
Error: Error linting charts: Error processing charts
------------------------------------------------------------------------------------------------------------------------
 ✖︎ icm-as => (version: "0.3.8", path: "charts/icm-as") > Error waiting for process: exit status 1
------------------------------------------------------------------------------------------------------------------------
Error linting charts: Error processing charts
Error: Process completed with exit code 1.
```

--- a/.github/workflows/lint-test_icm-as.yml
+++ b/.github/workflows/lint-test_icm-as.yml
@@ -35,12 +35,12 @@ jobs:
       - name: Run chart-testing (list-changed)
         id: list-changed
         run: |
-          changed=$(ct list-changed --config ct_icm.yaml)
+          changed=$(ct list-changed --config ct_icm-as.yaml)
           if [[ -n "$changed" ]]; then
             echo "::set-output name=changed::true"
           fi
       - name: Run chart-testing (lint)
-        run: ct lint --config ct_icm.yaml
+        run: ct lint --config ct_icm-as.yaml
 
       - name: Run Helm Unittest
         run: |

